### ISSUE TITLE
Build failed on Windows. Path concatenation doesn't work?

### ISSUE THREAD

**Issue by cai-lw** [koute/stdweb#185]
Failed to build stdweb on Windows with `nightly-x86_64-pc-windows-msvc`:
```
error: couldn't read "\\\\?\\D:\\projects\\screeps-ai-rust\\target\\wasm32-unknown-unknown\\release\\build\\stdweb-8ff62e980cfef1fa\\out/runtime.rs": 文件名、目录名或卷标语法不正确。 (os error 123)
  --> C:\Users\cai_l\.cargo\registry\src\github.com-1ecc6299db9ec823\stdweb-0.4.2\src\webcore\initialization.rs:19:5
   |
19 |     include!( concat!( env!( "OUT_DIR" ), "/runtime.rs" ) );
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
error: aborting due to previous error
error: Could not compile `stdweb`.
```

https://github.com/koute/stdweb/blob/39a99559596ada77c113ec49bdbb563b4a85f87c/src/webcore/initialization.rs#L19

**Comment by koute** [comment]
1. Are you using `cargo` directly or do you compile with `cargo-web`?
2. Can you try to modify the source and change `/runtime.rs` to `\\runtime.rs` and tell me if that fixes the issue?

**Comment by cai-lw** [comment]
@koute 
1. I was using `cargo web` to build other libraries that depends on `stdweb`.
2. When I build `stdweb` itself alone, it succeeded even without modifying. When building `stdweb` as dependency, changing to `\\runtime.rs` works.

**Comment by koute** [comment]
Hmm... I'm somewhat confused why this happens as other crates seem to do something similar and they're fine? (Maybe?)

Anyhow, I've added a workaround; can you grab the `stdweb` from `master` and tell me if it fixes the issue for you?

**Comment by cai-lw** [comment]
@koute Yes it worked now. Thank you.

**Comment by koute** [comment]
Great! This fix will be included in `0.4.3` which I plan to release this week.

Thanks for the report!
